services/common/workflow.js

```diff
@@ -1,10 +1,12 @@
+import { baseWaiver } from "./type/baseWaiver.js";
+
 export const ONEMAC_TYPE = {
   CHIP_SPA: "chipspa",
   CHIP_SPA_RAI: "chipsparai",
   MEDICAID_SPA: "medicaidspa",
   MEDICAID_SPA_RAI: "medicaidsparai",
-  WAIVER: "waiver",
-  WAIVER_BASE: "waivernew",
+  WAIVER: baseWaiver.componentType,
+  WAIVER_BASE: baseWaiver.componentType,
```

Comment:
Im suspicious that if these are the same do we really need both?

Reply:
Actually, that is the reason I said we should change Medicaid SPA everywhere at once as part of a story.... we use waiver and waivernew as the component type for the base waivers and should definitely clean that up someday :) 

I should probably revert that back to the text, though... I started down a path and it didn't pan out and that piece, while not wrong, is kinda weird to leave like that...